src/pages/index.tsx

```diff
@@ -1,9 +1,15 @@
+import { Text } from "@nextui-org/react";
+
 export default function Home() {
   return (
     <div>
-      <h1 className="text-3xl font-bold underline">
+      {/* Text is a NextUI component added to test the theme.ts file and for preview */}
+      <Text h1
```

Comment:
Didn't run Prettier on your source code? This section of the code doesn't appear to adhere to the standard code quality practices. So, make sure you run Prettier (manually if needed because I know the pre-commit hooks aren't working for you).